### PR TITLE
Use dancier color palette and a dummy menu and logo

### PR DESCRIPTION
**Pull request by manuela2013**
- Changed color palette
- Added dummy menu (not responsive yet)
- Added logo

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ import { SharedModule } from '@shared/shared.module';
 import { DefaultLayoutComponent } from './layout/default-layout/default-layout.component';
 import { NavigationComponent } from './layout/navigation/navigation.component';
 import { MatButtonModule } from '@angular/material/button';
+import {MatMenuModule} from "@angular/material/menu";
```
Comment (jans510) [inline src/app/app.module.ts:15]: please add whitespaces ;)

File: src/app/features/home/pages/landing-page/landing-page.component.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
 Finde bei uns
 
+
```
Comment (jans510) [inline src/app/features/home/pages/landing-page/landing-page.component.html:3]: I think the new lines are not needed :)

File: src/app/app.module.ts (hunk 1 above)
Comment (gorzala) [reply src/app/app.module.ts:15]: :+1:

Comment (jans510) [reply src/app/app.module.ts:15]: ... and single quotes. What do you think about using a linter?

Comment (gorzala) [reply src/app/app.module.ts:15]: @halbekanne didn*t we have a linter in this project? (I can remember something that prevents console.log() ;-) )

Comment (gorzala) [reply src/app/app.module.ts:15]: @halbekanne btw: excellent new username

Comment (jans510) [reply src/app/app.module.ts:15]: there is a eslintrc file, maybe it needs some extra rules...

Comment (gorzala) [reply src/app/app.module.ts:15]: @jans510 maybe you could be so nice...

Comment (jans510) [reply src/app/app.module.ts:15]: running ng lint shows also some console log statements ;) Should linting maybe be run on every commit as a hook?

Comment (gorzala) [reply src/app/app.module.ts:15]: I would expect this, and I also remember on a PR that contained some messages from eslint. @halbekanne did anything change here? Intentionally?

Comment (gorzala) [reply src/app/app.module.ts]: https://github.com/dancier/show-dancer/blob/master/.github/workflows/lint.yml

Comment (jans510) [reply src/app/app.module.ts]: but seem to not work as expected. By the way, you also have a prettier setup, so you could auto format on saving files

Comment (jans510) [reply src/app/app.module.ts]: it seem to just check files that were changed in the PR. What I not understand is, we have husky precommits defined to lint on commits but it does not work for me. Tried out a bit but weren't able to get husky to run.

Comment (manuela2013) [reply src/app/app.module.ts]: It was auto formatting of intellij. Maybe I need to adapt my intellij formatting rules as well.

File: src/app/features/home/pages/landing-page/landing-page.component.html (hunk 2 above)
Comment (manuela2013) [reply src/app/features/home/pages/landing-page/landing-page.component.html:3]: Why did I add it😅

**Comment by manuela2013** [comment]
@jans510 Nice! Now ng lint shows me more errors!

**Comment by gorzala** [comment]
:+1:

**Review by gorzala — APPROVED** [review]

**Pull request merged**